site/_apps/angular.txt

```diff
@@ -0,0 +1,14 @@
+---
+weight: 5
+title:  Angular
+logo: /assets/images/angular-logo.svg
+lighthouse: 100/100
+interactive-em: 5.8s
+interactive-faster-3g: 3.48s
+lighthouse-link: https://www.webpagetest.org/lighthouse.php?test=170423_BN_32H&run=1
+wpt-em-link: https://www.webpagetest.org/result/170420_46_67R/
+wpt-faster-3g-link: https://www.webpagetest.org/result/170328_N6_761d4f12fce28adc5163a3d056ce0af6/
+image: /assets/images/angular2hn-mobile.png
```

Comment:
I wonder if we can trim these down a little:

![image](https://cloud.githubusercontent.com/assets/110953/25319689/4d38cf44-2855-11e7-98aa-1f6ba0b8381e.png)

A quick pass-through ImageOptim seems like it'd save us about 35% on some of these:

![image](https://cloud.githubusercontent.com/assets/110953/25319729/ac160608-2855-11e7-8087-12bde6ace5ea.png)

I've also seen a ~55% win using https://compressor.io/compress. We can do those optimizations as part of this PR or work on it in a separate issue.